.readthedocs.yml

```diff
@@ -15,6 +15,8 @@ build:
 sphinx:
   builder: html
   configuration: docs/conf.py
+# might want to do this in future:
+  # fail_on_warning: true
```

Comment:
Do we want to do this? Delete these lines? Or keep it around for "future" still?

Reply:
Hmm, I'm not sure.  I think the reason it's the only change in that file is because there were other changes that were not different after a rebase. I can take out this change.